### PR TITLE
Added post about ML code generation vs coding by hand.

### PR DESCRIPTION
**Pull request by matijaSos**

### CODE REVIEW

File: web/blog/2022-06-24-ML-code-gen-vs-coding-by-hand-future.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+<ImgWithCaption
+    alt="Fizz Buzz with Copilot - stop"
+    source="img/fizz-buzz-copilot-stop.gif"
+    caption="The future is now, old man."
+/>
+
+
+\**Not to be biased, there are also other solutions offering similar functionality - e.g. [TabNine](https://www.tabnine.com/), [Webstorm has its own](https://www.jetbrains.com/help/webstorm/auto-completing-code.html#ml_completion), [Kite](https://www.kite.com/), [GPT Code Clippy (OSS attempt)](https://github.com/CodedotAl/gpt-code-clippy) et al., but Github Copilot recently made the biggest splash.*
+
+### Writing that informed this post
```
Comment (matijaSos) [inline web/blog/2022-06-24-ML-code-gen-vs-coding-by-hand-future.md:235]: @shayneczyzewski here it is! not sure why it hasn't showed up for you.

**Comment by matijaSos** [comment]
> Looking good, @matijaSos . Loaded it up and looks nice. Gave a quick read but didn't re-review content. Saw a reduction in memes too 🥲 looks like you still need to add to the bottom two sections though: `Recommended further reading`, and `Thanks to the reviewers`. Otherwise, lookin good 👍

hmm seems like you got the earlier version of the branch somehow, but I never pushed that one. In the commit, all the sections should be there.

**Comment by shayneczyzewski** [comment]
> > Looking good, @matijaSos . Loaded it up and looks nice. Gave a quick read but didn't re-review content. Saw a reduction in memes too 🥲 looks like you still need to add to the bottom two sections though: `Recommended further reading`, and `Thanks to the reviewers`. Otherwise, lookin good 👍
> 
> hmm seems like you got the earlier version of the branch somehow, but I never pushed that one. In the commit, all the sections should be there.

When I switched back to the branch this morning, it said:
```
Switched to branch 'matija-copilot-dsl-post'
Your branch and 'origin/matija-copilot-dsl-post' have diverged,
and have 1 and 1 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
```

So I guess they somehow diverged, but you probably made a force push to fix it recently, so I will try to look one more time! 👍🏻

File: web/blog/2022-06-24-ML-code-gen-vs-coding-by-hand-future.md (hunk 1 above)
Comment (shayneczyzewski) [reply web/blog/2022-06-24-ML-code-gen-vs-coding-by-hand-future.md:235]: Great, I see it now after doing a force reset to remote head! Looking good! 👍🏻 Lots of reviewers, I like it :D

**Review by shayneczyzewski — APPROVED** [review]

**Pull request merged**